mobile/src/screens/HomeScreen.tsx

```diff
@@ -19,6 +19,19 @@ export const HomeScreen: React.FC = () => {
     .slice(0, 5);
   const allBooks = booksCtx.books;
 
+  let newBooksRender;
+  let colBooksRender;
+  const booksLoaded: boolean = newBooks.length > 0;
```

Comment:
This feels kinda hacky. Can we add a `loading` bool to the `BookContext`/`BookState` type? That's a pretty common pattern in React data fetching. The idea being that `fetchBooks()` would set `loading` to true, fetch the books, then set `loading` to false. Then you can just do `booksCtx.loading`.